mne/preprocessing/maxwell.py

```diff
@@ -0,0 +1,530 @@
+# Authors: Mark Wronkiewicz <wronk.mark@gmail.com>
+#          Jussi Nurminen <jnu@iki.fi>
+
+
+# License: BSD (3-clause)
+
+from __future__ import division
+from os import path as op
+import warnings
+import numpy as np
+from scipy.linalg import pinv
+from scipy.misc import factorial
```

Comment:
You want to just operate on integers, right? It is cleaner (and maybe faster?) to use `from math import factorial`, then. The `scipy` one is more useful if you want to operate on `ndarray` or on `float` values.